app/controller/userController.js

```diff
@@ -126,8 +109,42 @@ class userController {
       return res.status(400).json(error);
     })
   }
+  static async ViewAllMentor(req, res) {
+    jwt.verify(req.token, process.env.JWT_KEY, async (err, authData) => {
+      if (err) {
+        res.sendStatus(403);
+      } else {
+        pool.on('error', (err, client) => {
+          console.error('Unexpected error on idle client', err)
+          process.exit(-1)
+        });
+        (async () => {
+          const client = await pool.connect()
```

Comment:
This should be done in models not controllers, Also move the `jwt.verify` to helpers